pkg/operator/starter.go

```diff
@@ -44,13 +45,6 @@ func RunOperator(ctx context.Context, controllerContext *controllercmd.Controlle
 	if err != nil {
 		return err
 	}
-	migrationClientConfig := dynamic.ConfigFor(controllerContext.KubeConfig)
-	migrationClientConfig.Burst = 40
-	migrationClientConfig.QPS = 30
-	dynamicClientForMigration, err := dynamic.NewForConfig(migrationClientConfig)
```

Comment:
verify that kube-storage-version-migrator has similar values.

Reply:
kube-storage-version migrator does not set anything specific. so, i restored these values in new commits.